Removed chapter about exporting default messages. (#1860)
cap-js/docs: java/event-handlers/indicating-errors.md

The error messages are optimized for UI scenarios and avoid any technical references to entity names or element names. Message targets are used where appropriate to allow the UI to show the error message next to the affected UI element.
You can enable these translated error messages by setting [<Config java>cds.errors.defaultTranslations.enabled: true</Config>](../developing-applications/properties#cds-errors-defaultTranslations-enabled).

### Provide custom error messages

> By default, CAP Java provides error messages in several languages. If these error messages or translation are not sufficient for an application, they can be overwritten with custom error messages. Applications can overwrite these texts by providing new error messages under the respective error code in the application's `messages.properties` resource bundle under `src/main/resources`.

Reviewer: ```suggestion
By default, CAP Java provides error messages in several languages. If an error message or translation isn't sufficient for an application, it can be overwritten with a custom error message. Applications can provide the new error message under the respective error code in the application's `messages.properties` resource bundle under `src/main/resources`.
```

The error messages are optimized for UI scenarios and avoid any technical references to entity names or element names. Message targets are used where appropriate to allow the UI to show the error message next to the affected UI element.
You can enable these translated error messages by setting [<Config java>cds.errors.defaultTranslations.enabled: true</Config>](../developing-applications/properties#cds-errors-defaultTranslations-enabled).

### Provide custom error messages

> By default, CAP Java provides error messages in several languages. If these error messages or translation are not sufficient for an application, they can be overwritten with custom error messages. Therefore, an application can provide a `messages_<LOCALE>.proberties` resource bundle in the folder `src/main/resources/`.

Reviewer 2: ```suggestion
By default, CAP Java provides error messages in several languages. If these error messages or translation are not sufficient for an application, they can be overwritten with custom error messages. Applications can overwrite these texts by providing new error messages under the respective error code in the application's `messages.properties` resource bundle under `src/main/resources`.
```

> By default, CAP Java provides error messages in several languages. If these error messages or translation are not sufficient for an application, they can be overwritten with custom error messages. Therefore, an application has to provide a `errors_<LOCALE>.proberties` resource bundle in the folder `srv/src/main/resources/com/sap/cds/i18n/`.

Reviewer 2: No, error messages should be maintained as part of the standard `messages.properties` file of the application. The `errors.properties` file in the `com/sap/cds/i18n` should not be messed with / overwritten. Application provided texts in the standard message bundles of the app always have precedence.

Author: done

> To know which error codes and messages are available by default, one can download the artifact `cds-services-impl-<REVISION>-sources.jar` from the public Maven repository. Then extract the Java enumeration `com.sap.cds.services.utils.CdsErrorStatuses` from this source jar. This enumartion shows all available error codes and messages that are used by the CAP Java runtime.

Reviewer 2: I'd mention that one can look at the `CdsErrorStatuses` class preferrably via an IDE,  which is most likely the easiest thing to do. Only if that would not be possible for some reason you could also mention to download the sources jar directly.

Author: done

The error messages are optimized for UI scenarios and avoid any technical references to entity names or element names. Message targets are used where appropriate to allow the UI to show the error message next to the affected UI element.
You can enable these translated error messages by setting [<Config java>cds.errors.defaultTranslations.enabled: true</Config>](../developing-applications/properties#cds-errors-defaultTranslations-enabled).

> ### Exporting the Default Messages

Reviewer 2: I think we should describe how stakeholders can find information how to replace dedicated error messages instead, if that bundle no longer exists.

Author: done

Reviewer 2: I wouldn't maintain two different explanations
